edly-io/h5pxblock
# feat: add support for due dates

### Description

This PR makes the user unable to store their progress in the XBlock if the due date has passed.

### Testing instructions

1. Set the due date for a subsection with an h5p content to any date in the past
![image](https://github.com/edly-io/h5pxblock/assets/33465240/8e6d4f68-5b23-4e0b-85e1-3f90dcc030d6)
2. As a student, verify he can interact with the h5p content but it's progress and grades are not stored

After:
![image](https://github.com/edly-io/h5pxblock/assets/33465240/f289b47e-aaa3-4d89-a487-0c91f13b75ed)
